request_a_govuk_domain/request/utils.py

```diff
@@ -19,10 +23,22 @@ def handle_uploaded_file(file):
     with open(file_path, "wb") as destination:
         for chunk in file.chunks():
             destination.write(chunk)
-
     return saved_filename
 
 
+def validate_file_infection(file):
+    """
+    Incoming file is sent to clamd for scanning.
+    Raises a ValidationError
+    """
+
+    cd = clamd.ClamdNetworkSocket(settings.CLAMD_TCP_ADDR, settings.CLAMD_TCP_SOCKET)
+    result = cd.instream(file)
+
+    if result and result["stream"][0] == "FOUND":
+        raise ValidationError("File is infected with malware.", code="infected")
```

Comment:
I think we need a custom exception, since this is a lot more important than just validation and will probably be treated differently